databases/week1/week1.sql

```diff
@@ -0,0 +1,60 @@
+--Find out how many tasks are in the task table
+SELECT COUNT(id) as total_task FROM task;
+
+--Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(id)
+FROM task
+WHERE due_date IS NULL;
+
+--Find all the tasks that are marked as done
+SELECT * 
+FROM task JOIN status
+ON task.status_id = status_id
```

Comment:
Join statement and join condition are normally written on their specific line
pointer: improves query readability

Reply:
please see other queries where this happens.